test/perf/workloads/small-messages-burst.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+echo "Creating pipeline..."
+PIPELINE_ID=$(
+curl -Ss -X POST 'http://localhost:8080/v1/pipelines' -d '
+{
+    "config":
+    {
+        "name": "perf-test",
+        "description": "Test performance"
+    }
+}' | jq -r '.id'
+)
+
+
+echo "Creating a normal source..."
+NORMAL_SOURCE=$(
+jq -n --arg pipeline_id "$PIPELINE_ID" '{
+    "type": "TYPE_SOURCE",
+    "plugin": "builtin:generator",
+    "pipeline_id": $pipeline_id,
+    "config":
+    {
+        "name": "normal-source",
+        "settings":
+        {
+            "fields": "id:int,name:string,company:string,trial:bool",
+            "format": "structured",
+            "readTime": "100ms",
+            "recordCount": "-1"
```

Comment:
This fields are not correct as of a couple of minutes ago, due to some generator changes. I'll change this soon.